packages/docs/components/sidebar-logo.tsx

```diff
@@ -0,0 +1,31 @@
+"use client";
+
+import Image from "next/image";
+import { useState } from "react";
+
+interface SidebarLogoProps {
+  src: string;
+  alt: string;
+  width?: number;
+  height?: number;
+  className?: string;
+}
```

Comment:
This interface isn't needed, as `ImageProps` exists

```suggestion
import Image, { type ImageProps } from "next/image";
import { useState } from "react";
```